daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -59,6 +61,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
 | host               | Y        | Connection-string for the rabbitmq host  | `amqp://user:pass@localhost:5672`
+| consumerID         | N        | Consumer identifier  |
```

Comment:
@dmitsh  - Can you add a few more words here on the consumer identifier, what this is used for? Will this be autogenerated by Dapr if not provided? Also can you provide a example in the example column?

Reply:
@msfussell , please review the wording.